File: livingrimoire start here/LivinGrimoire java/src/Auxiliary_Modules/AXCmdBreaker.java
package Auxiliary_Modules;

public class AXCmdBreaker {
    // separate command parameter from the command
    public String conjuration;

    public AXCmdBreaker(String conjuration) {
        this.conjuration = conjuration;
    }
    public String extractCmdParam(String s1){
        if (s1.contains(conjuration)){
            return s1.replace(conjuration,"").trim();
        }
        return "";
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/AXCmdBreaker.kt
package auxiliary_modules

class AXCmdBreaker( // separate command parameter from the command
    var conjuration: String
) {
    fun extractCmdParam(s1: String): String {
        return if (s1.contains(conjuration)) {
            s1.replace(conjuration, "").trim { it <= ' ' }
        } else ""
    }
}